src/site/content/en/blog/badging-api/index.md

```diff
@@ -126,108 +98,58 @@ and users must add it to their home screens.
 
 The Badge API consists of the following methods on `navigator`:
 
-* `setExperimentalAppBadge([number])`: Sets the app's badge. If a value is
+* `setAppBadge([number])`: Sets the app's badge. If a value is
   provided, set the badge to the provided value otherwise, display a plain
   white dot (or other flag as appropriate to the platform).
-* `clearExperimentalAppBadge()`: Removes app's badge.
+* `clearAppBadge()`: Removes app's badge.
 
 For example:
 
 ```js
 // Set the badge
 const unreadCount = 24;
-navigator.setExperimentalAppBadge(unreadCount);
+navigator.setAppBadge(unreadCount);
 
 // Clear the badge
-navigator.clearExperimentalAppBadge();
+navigator.clearAppBadge();
 ```
 
-`setExperimentalAppBadge()` and `clearExperimentalAppBadge()` can be called from
-a foreground page, or potentially in the future, a service worker. In either
-case, it affects the whole app, not just the current page.
+You can only call `setAppBadge()` and `clearAppBadge()`
+from a foreground page, or potentially in the future, a service worker. In
+either case, it affects the whole app, not just the current page.
 
 In some cases, the OS may not allow the exact representation of the badge.
 In such cases, the browser will attempt to provide the best representation for
 that device. For example, because the Badging API isn't supported on Android,
 Android only ever shows a dot instead of a numeric value.
 
-Don't assume anything about how the user agent wants to display the badge.
+Don't assume anything about how the user agent displays the badge.
 Some user agents may take a number like "4000" and rewrite it as
 "99+". If you saturate the badge yourself (for example by setting it to "99")
 then the "+" won't appear. No matter the actual number, just call
-`setExperimentalAppBadge(unreadCount)` and let the user agent deal with
+`setAppBadge(unreadCount)` and let the user agent deal with
 displaying it accordingly.
 
 While the Badging API *in Chrome* requires an installed app, you shouldn't
 make calls to the Badging API dependent on the install state. Just call the
 API when it exists, as other browsers may show the badge in other places.
 If it works, it works. If not, it simply doesn't.
 
```

Comment:
I think it's premature to remove the "Supporting both the first and second origin trial" sections, since a similar thing applies here. Perhaps remove the first origin trial since that's long gone. "Supporting both the origin trial and released versions".

Remove the else if clauses for "window.ExperimentalBadge" since that's the first origin trial. And instead, do "if (navigator.setAppBadge) { ... } else if (navigator.setExperimentalAppBadge) { ... }".

Reply:
We're also still happy to hear feedback about the API design, though we don't want to make major incompatible changes, we can still add functionality.

Reply:
@jpmedley - can we add the section on supporting both back then please?

Reply:
@mgiuca Something's not making sense to me. OT tokens are deactivated when the OT ends. Are you telling me that tokens for the second trial are still being honored?

Reply:
Yes, tokens from the second trial are still being honored _on Chrome 80 and earlier_. Sites will want to continue working on Chrome 80 which still uses the `setExperimentalAppBadge` API, and this is a way to let them do that.

Reply:
Readding the feedback section.

Pardon me for being a pest, it seems lie you wouldn't want to issue new tokens at this point.

Reply:
If a user doesn't upgrade and stays on Chrome 80, the existing token will continue to function for some period of time.

Reply:
If that's your suggested rewrite, I agree.